ui/components/app/assets/asset-list/asset-list-control-bar/asset-list-control-bar.tsx

```diff
@@ -303,13 +310,38 @@ const AssetListControlBar = ({
     }
 
     return t('popularNetworks');
-  }, [isTokenNetworkFilterEqualCurrentNetwork, currentMultichainNetwork, t]);
+  }, [
+    enabledNetworksByNamespace,
+    isTokenNetworkFilterEqualCurrentNetwork,
+    currentMultichainNetwork.isEvmNetwork,
+    currentMultichainNetwork.network.nickname,
+    currentMultichainNetwork?.nickname,
+    t,
+    allNetworks,
+  ]);
+
+  const singleNetworkIconUrl = useMemo(() => {
```

Comment:
memo val to validate and grab network image for the network dropdown.

<img width="229" height="70" alt="Screenshot 2025-08-20 at 21 59 53" src="https://github.com/user-attachments/assets/0a1ecd49-c294-4f09-9597-cc90db7df539" />
<img width="174" height="67" alt="Screenshot 2025-08-20 at 21 59 58" src="https://github.com/user-attachments/assets/2567d409-a160-42ba-a62f-76cc12db6b39" />